ext/js/background/backend.js

```diff
@@ -1303,7 +1327,7 @@ export class Backend {
 
         this._mecab.setEnabled(options.parsing.enableMecabParser && enabled);
 
-        if (options.clipboard.enableBackgroundMonitor && enabled) {
+        if ((options.clipboard.enableBackgroundMonitor || options.clipboard.enableSearchPageMonitor) && enabled) {
```

Comment:
This change seems fine and good but I'm not sure about the rest of this pr. Can't replicate the issue you're mentioning at all. 

This bit "No longer creates a new search window everytime clipboard changes (unless background clipboard text monitoring is also enabled)." seems to already be how Yomitan works. 

I would be okay with approving and getting this merged with only this one line changed. Maybe split the rest into a separate pr and try to provide some steps for reproducing the issue.

Reply:
> This bit "No longer creates a new search window everytime clipboard changes (unless background clipboard text monitoring is also enabled)." seems to already be how Yomitan works.
>
That is actually not true. I just tested it by cloning and **only adding the if statement part** and it does in fact create a new popup every time with only the `search page clipboard text monitoring` being enabled, it's only supposed to update the search page **if one is open**, not create a new one _(unless `background clipboard text monitoring`)_ is enabled. 

Not only that, even with that if statement, `background clipboard text monitoring` does all the work, you don't even need to have `search page clipboard text monitoring` enabled for it to update the search page, so basically it's useless. 

In my pr, the `enable cb monitor if either cb setting === true` commit **does not disable making a brand new popup every time if one is not open** _(which is the main problem)_, it only give it permission to update the search without needing the first option to be enabled. 

> Is there a way to open that special search page without having to enable `Enable background clipboard text monitoring` first?

Yes, under `Native Keyboard Shortcuts` -> `Open the popup window`. But that doesn't change anything, it still creates a new popup every time, because the logic in `backend.js` has no other choice. That's the whole reason I made the ghost tab.

The rest of my pr allows the second option, instead of creating a new search tab popup, to create a fake tab if none exist, or update the search page if one does.

I believe this was the intended feature of that second option..? However as I mentioned before, it currently does nothing...

> I would be okay with approving and getting this merged with only this one line changed. Maybe split the rest into a separate pr and try to provide some steps for reproducing the issue.

As for this, there's not much else I can do on my end, If you have doubts I can go and get some more users to go upvote / give further evidence in the issue that I made.

As I showed in the video, I am on the latest version + chrome (*tried right now, it still doesn't work for me*), with default settings, and also by cloning this repo and only adding that if statement (default settings). Same issue, that video is still accurate as of right now.

I would like more developers to test this before my pr gets discarded, as **I can't be the only person experiencing this issue, that doesn't make any sense**.

I will try to get some regular users to try it and see what's going on for them as well. 👍🏼

Reply:
I just tested this again to sanity check and yeah just including this one line doesn't look like a good idea. I still can't replicate any of the other issues though.

I think the issue you're having needs a step by step reproduction process starting from Yomitan set to default settings.

Reply:
### Steps
1. Reset Yomitan settings.
2. **Only** Turn on **`search page`** clipboard text monitoring.
<img src="https://github.com/themoeway/yomitan/assets/106574385/2d19a974-c193-4c46-bb6a-87c6999463c4" width="300" />

3. Copy a word to clipboard... -> **Nothing happens**. 
4. Turn off **`search page`** clipboard text monitoring.
5. **Only** turn on **`background`** clipboard text monitoring`.
<img src="https://github.com/themoeway/yomitan/assets/106574385/76d4d58a-a52c-4e38-84c1-9c10d7179e51" width="300" />

6. Copy a word to clipboard... -> **Search popup gets updated**. 

### Chrome Extension Video Details `5/27/2024`.
- **Yomitan `v24.5.14.1`** from Google Chrome Store (Default Settings).
- **Google Chrome `v125.0.6422.112`** (Official Build) (64 bit)
#### Also Tested on
- **Microsoft Edge `v126.0.2592.13`** (Official Build) beta (64 bit)

### Chrome Extension Video Evidence
[video](https://github.com/themoeway/yomitan/assets/106574385/654b437e-7335-4142-bb6d-809c51edd46a)

- I also tested this by cloning the Github repo, it does the exact same thing shown in the Chrome  Extension Video.